### DISCUSSION TITLE
TAB key on grep_project behaves differently from find_files

### DISCUSSION THREAD

**Discussion by estacioneto** [ibhagwan/fzf-lua#1411] [Q&A] [answered]
Hi and thanks again for the amazing plugin!

I'm trying to have the same behavior I have on `find_files` of pressing TAB and selecting an entry when using `grep_project` but I believe what it does is reloading the results, right?

I couldn't find the discussion about it (even though I've found #1170 and #1171) so I have some questions:

1. How can I do it on my end? Can I do it by changing something in the config?
2. Should it be the default behavior for both or should they behave differently?
3. Should we leave it like it is and have another action keymap like the `<C-x>` or similar (since it's already used to close buffers) to specifically deal with that across pickers?

Again, thank you so much!

**Comment by ibhagwan** [comment]
> I'm trying to have the same behavior I have on find_files of pressing TAB and selecting an entry when using grep_project but I believe what it does is reloading the results, right?

I’m not sure what you mean, by default `Tab` is used to select items and should behave the same on both files and grep, if this isn’t the case in your system it might be your fzf-lua config or your shell’s `$FZF_DEFAULT_OPTS`, you can rule out the former by using `mini.sh`, you can rule out the latter by setting `:lua vim.g.FZF_DEFAULT_OPTS=''`.

Running minimal config with mini.sh
```sh
sh -c "$(curl -s https://raw.githubusercontent.com/ibhagwan/fzf-lua/main/scripts/mini.sh)"
```

**Reply by estacioneto** [reply]
Oh, thank you for that `mini.sh` explanation! I'll definitely use it when checking if it's something on my side or not.

It's weird that `vim.g.FZF_DEFAULT_OPTS` is `nil` and `$FZF_DEFAULT_OPTS` is empty. Do you know something else that might be causing this?

[Here's my config](https://github.com/estacioneto/dotvim/blob/master/lua/estacio/fzf/init.lua). And also a video showcasing what happens on my end

https://github.com/user-attachments/assets/e34ca1a3-394b-40f3-a477-14086cc94c09

**Comment by ibhagwan** [comment, marked as answer]
https://github.com/estacioneto/dotvim/blob/7aa4ecd63750a039f896544c599ddd7065fe96a9/lua/estacio/fzf/init.lua#L148

This is your issue, `ctrl-i` is equivalent to tab, change it to something else.

**Reply by estacioneto** [reply]
> `ctrl-i` is equivalent to tab

🤯🤯🤯

It worked. Thank you!
